sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/models/RetentionPolicy.java

```diff
@@ -5,32 +5,39 @@
 package com.azure.data.tables.implementation.models;
 
 import com.azure.core.annotation.Fluent;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+import com.azure.core.util.CoreUtils;
+import com.azure.xml.XmlReader;
+import com.azure.xml.XmlSerializable;
+import com.azure.xml.XmlToken;
+import com.azure.xml.XmlWriter;
+import javax.xml.namespace.QName;
+import javax.xml.stream.XMLStreamException;
 
-/** The retention policy. */
-@JacksonXmlRootElement(localName = "RetentionPolicy")
+/**
+ * The retention policy.
+ */
 @Fluent
-public final class RetentionPolicy {
+public final class RetentionPolicy implements XmlSerializable<RetentionPolicy> {
     /*
      * Indicates whether a retention policy is enabled for the service.
      */
-    @JsonProperty(value = "Enabled", required = true)
```

Comment:
I noticed thre is a "required" to be true here... but I don't see any logic in the toXml and fromXml. Am i missing something?

Reply:
This is in regard to not checking that `Enabled` was found, right?

I think the old annotations were wrong as the configurations for Autorest Java don't include the flag for marking required properties as required. Which is why it doesn't show up in the `toXml` or `fromXml` methods.